src/components/BusinessReviews/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "../../theme/colors.scss";
+.rating-container {
+  display: flex;
+  padding: 10px;
+  justify-content: center;
+}
+.MuiLinearProgress-root {
+  width: 200px;
+  display: flex;
+  align-self: center;
+  margin: 5px;
+  height: 6px;
+}
+.MuiLinearProgress-barColorPrimary {
+  background-color: $light-green !important;
+}
+.MuiLinearProgress-colorPrimary {
```

Comment:
hmm there must be a better way to change the styling of the `LinearProgress` component no ?